scala/lambdas/ingest-asset-reconciler/src/main/scala/uk/gov/nationalarchives/ingestassetreconciler/Lambda.scala

```diff
@@ -64,6 +65,15 @@ class Lambda extends LambdaRunner[Input, StateOutput, Config, Dependencies] {
       titleOfCoWithoutExtension == assetChildTitleOrFileNameWithoutExtension
     }
 
+  private def doesChecksumMatchFixity(item: DynamoFormatters.FileDynamoTable, fixities: List[Client.Fixity]): Boolean = {
+    val sortedItems = item.checksums.sortBy(_.algorithm)
+    val sortedFixities = fixities.sortBy(_.value)
```

Comment:
How come this is sorted by `value` instead of `algorithm`?

Reply:
Arghhh, well spotted, I'll correct it, I guess the tests passed just because the values are ACCIDENTALLY in same sorted order as the sorting order  of algorithm.